CHANGELOG.adoc

```diff
@@ -111,7 +111,27 @@ They will be displayed if a project template with one of the following specific
 Thus, downstream projects that would want to hide these cards, will not be able to do it by not contributing them to the `createProjectAreaCardExtensionPoint` extension point, but by providing a `ICapabilityVoter`.
 Another consequence is that the `projectTemplates` graphql query now requires the `context` parameter.
 A new "demo" profile is now available to start Sirius Web as a demo instance.
+- https://github.com/eclipse-sirius/sirius-web/issues/5126[#5126] [core] Remove `readonly` props from EditProjectNavbar since this props was only used to enable or disable the project rename button.
+This purpose can be achieved with the `canRename` capabilities.
+- https://github.com/eclipse-sirius/sirius-web/issues/5126[#5126] [sirius-web] Remove the `editProjectViewReadOnlyPredicateExtensionPoint` that was use to define whether the edit project view should be read-only or not.
+This purpose can be achieve with the `canEdit` project capabilities by contributing a `ICapabilityVoter`:
+[source,java]
+----
+@Service
+public class canEditProjectCapabilityVoter implements ICapabilityVoter {
+
+  public CapabilityVote vote(String type, String identifier, String capability) {
+    if ("Project".equals(type) && "canEdit".equals(capability) && identifier != null) {
```

Comment:
Use the constants since it's not "canEdit" for example but "edit"

Reply:
Done